docs/src/apps/jsonrpc-api.md

```diff
@@ -1016,6 +1020,118 @@ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "
+
+```bash
+// Request
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getTokenAccountBalance", "params": ["7fUAJdStEuGbc3sM84cKRL6yYaaSstyLSU4ve5oovLS7", "TokenSVp5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o"]}' http://localhost:8899
+// Result
+{"jsonrpc":"2.0","result":{"context":{"slot":1114},"value":9864,"id":1}
+```
+
+### getTokenAccountsByDelegate
+
+Returns all SPL Token accounts by approved Delegate.
+
+#### Parameters:
+
+- `<string>` - Pubkey of account delegate to query, as base-58 encoded string
+- `<string>` - Pubkey of Token program ID that owns the accounts, as base-58 encoded string
+- `<object>` - (optional) Configuration object containing the following optional fields:
+  * (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+  * (optional) `mint: <string>` - specific token Mint to limit accounts to
+
+#### Results:
+
+The result will be an RpcResponse JSON object with `value` equal to an array of JSON objects, which will contain:
+
+- `pubkey: <string>` - the account Pubkey as base-58 encoded string
+- `account: <object>` - a JSON object, with the following sub fields:
+   - `lamports: <u64>`, number of lamports assigned to this account, as a u64
+   - `owner: <string>`, base-58 encoded Pubkey of the program this account has been assigned to
+   `data: <object>`, Token state data associated with the account, in JSON format `{<program>: <state>}`
+   - `executable: <bool>`, boolean indicating if the account contains a program \(and is strictly read-only\)
+   - `rentEpoch: <u64>`, the epoch at which this account will next owe rent, as u64
+
+#### Example:
+
+```bash
+// Request
+curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc":"2.0", "id":1, "method":"getTokenAccountsByDelegate", "params": ["4Nd1mBQtrMJVYVfKf2PJy9NZUZdTAsp7D4xWLs4gDB4T", "TokenSVp5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o", {"mint":"3wyAj7Rt1TWVPZVteFJPLa26JmLvdb1CAKEFZm3NY75E"}]}' http://localhost:8899
+// Result
+{"jsonrpc":"2.0","result":{"context":{"slot":1114},"value":[{"data":{"token":{"account":{"amount":1,"delegate":"4Nd1mBQtrMJVYVfKf2PJy9NZUZdTAsp7D4xWLs4gDB4T","delegatedAmount":1,"isInitialized":true,"isNative":false,"mint":"3wyAj7Rt1TWVPZVteFJPLa26JmLvdb1CAKEFZm3NY75E","owner":"11111111111111111111111111111111"}}},"executable":false,"lamports":1726080,"owner":"TokenSVp5gheXUvJ6jGWGeCsgPKgnE3YgdGKRVCMY9o","rentEpoch":4},"pubkey":"CnPoSPKXu7wJqxe59Fs72tkBeALovhsCxYeFwPCQH9TD"}],"id":1}
+```
+
+### getTokenAccountsByOwner
+
+Returns all SPL Token accounts by token owner.
+
+#### Parameters:
+
+- `<string>` - Pubkey of account owner to query, as base-58 encoded string
+- `<string>` - Pubkey of Token program ID that owns the accounts, as base-58 encoded string
+- `<object>` - (optional) Configuration object containing the following optional fields:
+  * (optional) [Commitment](jsonrpc-api.md#configuring-state-commitment)
+  * (optional) `mint: <string>` - specific token Mint to limit accounts to
```

Comment:
Alternately, we could either (a) make this parameter required and only return accounts of a particular Mint; or (b) remove this parameter and return all accounts of any Mint that share the same owner. Thoughts?

Reply:
I like the current approach.  Same nit about the `mint: <string>` docs as above though